siliconcompiler/design.py

```diff
@@ -0,0 +1,535 @@
+import re
+from pathlib import Path
+from typing import List
+from siliconcompiler.schema import NamedSchema
+from siliconcompiler.schema import EditableSchema, Parameter, Scope
+from siliconcompiler.schema.utils import trim
+from siliconcompiler import utils
+from siliconcompiler import SiliconCompilerError
+
+
+###########################################################################
+class DesignSchema(NamedSchema):
+
+    def __init__(self, name: str):
+        NamedSchema.__init__(self, name=name)
+        schema_design(self)
+        # TODO: move this into use schema?
+        self.__dependency = {}
```

Comment:
I think you can safely remove this since we'll implement this separately and inherit it

Reply:
done